Sources/FoundationNetworking/CMakeLists.txt

```diff
@@ -1,3 +1,19 @@
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  find_package(CURL CONFIG)
+  if(NOT CURL_FOUND)
+    find_package(CURL REQUIRED)
+  endif()
+endif()
+
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  if((NS_CURL_ASSUME_FEATURES_MISSING) OR (CURL_VERSION_STRING VERSION_LESS "7.32.0"))
+    add_compile_definitions(NS_CURL_MISSING_XFERINFOFUNCTION)
+  endif()
+
+  if((NS_CURL_ASSUME_FEATURES_MISSING) OR (CURL_VERSION_STRING VERSION_LESS "7.30.0")) add_compile_definitions(NS_CURL_MISSING_MAX_HOST_CONNECTIONS)
```

Comment:
This is missing a line break. The GitHub split view makes this a bit hard to spot, but the `add_compile_definitions` is on the same line as the `if`, which causes a parse error in cmake.

Reply:
Word wrap has defeated me. Fixing shortly.